sm/plugins/tedge_docker_plugin/tedge_docker_plugin.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
```

Comment:
I feel below format is more readable.
 #!/bin/sh
main() {
    if [ "$1" = yes ]; then
        do_task_this
    else
        do_task_that
    fi
}

do_task_this() {
    ...
} 
do_task_that() {
    ...
}

Reply:
Are you suggesting to replace the switch case statements with `if-else` blocks? Or extracting the steps in the case blocks to dedicated methods? I'm not a big fan of the former. But the latter can be done.

Reply:
The current script is readable with an usual structure. There is no reason to change it.